### DISCUSSION TITLE
How to use React-Native-FireBase in Expo Project?

### DISCUSSION THREAD

**Discussion by AnatuGreen** [invertase/react-native-firebase#6015] [Q&A]
I initiated my project using expo.  I tried following the tutorial and docs but got stuck here: https://rnfirebase.io/#2-android-setup

My expo project has an android folder but the only contents are README.md and settings.json so there was no way to proceed with this documentation.

Is it that this module only works with bare react native app or is there a way to use this with expo?

**Comment by mikehardy** [comment]
I  believe this will help? https://rnfirebase.io/#expo
react-native-firebase implements Expos config plugins so it is possible in some scenarios.

**Reply by AnatuGreen** [reply]
Ok I am checking it. Thanks
